libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -342,6 +342,16 @@ export const fcmConfig: IConfigCredentials[] = [
   ...pushConfigBase,
 ];
 
+export const expoConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.ApiKey,
+    displayName: 'Access Token',
+    type: 'text',
+    required: true,
+  },
```

Comment:
@psteinroe is the access token should be secret? If so maybe better to add the secret flag here.

Reply:
I assumed that any API Key is flagged as secret because its listed in the secure-credentials:

```ts
export const secureCredentials: CredentialsKeyEnum[] = [
  CredentialsKeyEnum.ApiKey,
  CredentialsKeyEnum.SecretKey,
  CredentialsKeyEnum.Token,
  CredentialsKeyEnum.Password,
  CredentialsKeyEnum.ServiceAccount,
];
```

Either way, the `IConfigCredentials` interface does not have a secret flag. Could you specify?